examples/transports/README.md

```diff
@@ -132,8 +150,15 @@ node1.peerStore.addressBook.set(node2.peerId, node2.multiaddrs)
   )
 })();
 ```
+This code illustrates two nodes communicating with each other. Using `createNode()`, we create one node `node1` and another `node2`. We then make `node2` print any messages it receives on the `/print` protocol. Whenever a stream is opened on the `/print` protocol, the function supplied to [`handle`](https://github.com/libp2p/js-libp2p/blob/master/doc/API.md#handle]) is called.  Messages that arrive at `node2` come as a stream of data which we then `pipe` into `concat` so that the final string printed looks appropriately formatted. Next, we want to add `node2` into `node1`'s "address book" so that `node1` can contact `node2` via `node2`'s peer id (this will be demonstrated later). Think of the address book like a phone book where the associated peer id is like a friend's name and the corresponding set of multiaddresses is like a list of various phone numbers we could use to contact the friend. We now want `node1` to open a connection to `node2`. In libp2p, [this is known as dialing](https://docs.libp2p.io/reference/glossary/#dial). [Dialing requires that we specify both the peer being connected to and the protocol](https://github.com/libp2p/js-libp2p/blob/master/doc/API.md#dialprotocol). In this case, we use `node2`'s `peerId` to specify `node2` as the peer and state that `node1` will be dialing on the `/print` protocol. Note that because we used the `peerId` to identify `node2`, `node2` must be in `node1`'s address book. Omitting the line where we add `node2` into `node1`'s address book will generate an error. Alternatively, you could instead supply one of `node2`'s multiaddresses to dial `node2` similar to how you might call a friend using their phone number as opposed to referring to your contact list
```

Comment:
While this explanation is nice, I think we should not go this way. We are mixing examples with tutorials/docs and adding a lot of content for an example that should be simple. I'd rather see links to [libp2p/js-libp2p/blob/master/src/peer-store/README.md](https://github.com/libp2p/js-libp2p/blob/master/src/peer-store/README.md) and docs.libp2p.io explaining the process, so that we avoid duplicating documentation on how things work. Otherwise, we will be prone to get outdated information when we have it in not common places.